src/ripple/protocol/impl/InnerObjectFormats.cpp

```diff
@@ -143,6 +143,21 @@ InnerObjectFormats::InnerObjectFormats()
             {sfPublicKey, soeREQUIRED},
             {sfAccount, soeOPTIONAL},
         });
+
+    add(sfAmountEntry.jsonName.c_str(),
+        sfAmountEntry.getCode(),
+        {
+            {sfAmount, soeREQUIRED},
+            {sfFlags, soeOPTIONAL},
```

Comment:
What is it used for?

Reply:
Informing a 3rd party

Reply:
I read that wrong, inform is for 3rd party. The flag here is unused. I think we were discussing the option and never removed it. I will remove it now.